aiozk/recipes/base_lock.py

```diff
@@ -15,7 +15,18 @@ class BaseLock(SequentialRecipe):
         if timeout is not None:
             time_limit = time.time() + timeout
 
-        await self.create_unique_znode(znode_label)
+        try:
+            await self.create_unique_znode(znode_label)
+        except Exception as ex:
+            log.debug('unknown exception: %s', ex)
+            while True:
+                try:
+                    await self.delete_garbage_znodes(znode_label)
+                except Exception as e:
+                    log.debug('unknown exception: %s', e)
```

Comment:
This and upper log can be replaced with `log.exception` in that case it will print pretty traceback on it's own

Reply:
I run test with containers built on top of fan0/python:2.1.0. But that includes python version 3.7.5. How can I run pytest with python version 3.8?

Reply:
@rhdxmr I just run them in virtualenv from python3.8
```
docker run --name zookeeper -p 2181:2181 zookeeper
export ZK_HOST=localhost
./venv/bin/pytest aiozk/test/test_lock.py -s --log-cli-level DEBUG
```

Reply:
@cybergrind I installed python 3.8.1 and ran test code. But the error you mentioned is not reproduced in my machine.

> ERROR    asyncio:base_events.py:1692 An open stream object is being garbage collected; call "stream.close()" explicitly.

Reply:
I replaced `log.debug` with `log.exception`.
It is very useful to print stack trace of unknown exceptions. Thanks.